systems/subscriber/sim-manager/pkg/server/sim_manager.go

```diff
@@ -125,6 +123,9 @@ func (s *SimManagerServer) AllocateSim(ctx context.Context, req *pb.AllocateSimR
 	}
 
 	if req.SimToken != "" {
+		log.Info("sim token", req.SimToken)
```

Comment:
This line was for debugging purposes. we can remove it.

Reply:
addressed